src/components/SideBar/index.tsx

```diff
@@ -2,7 +2,7 @@ import Image from 'next/image';
 
 export default function SideBar() {
   return (
-    <div className='h-full w-[15%] border-r-[1px] border-[#ccc] px-2 py-4'>
+    <div className='h-full w-[15%] cursor-pointer border-r-[1px] border-[#ccc] px-2 py-4'>
```

Comment:
The hover styling should not apply to the whole the sidebar `div`, but its items.